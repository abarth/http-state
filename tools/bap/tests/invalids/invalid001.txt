; invalid test, just a string
test

