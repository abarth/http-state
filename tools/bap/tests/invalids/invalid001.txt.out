invalids/invalid001.txt(2:0): error: Got EOL, expecting '=' or '=/'
parsing failed: 1 errors encountered
